source/includes/api/_dashboard.md

```diff
@@ -107,7 +107,44 @@ curl -X GET https://api.resourcewatch.org/v1/dashboard?page[size]=15&page[number
 
 ### Sorting
 
-There's currently no support for custom sorting of dashboards. The dashboards are listed on a pseudo-random order.
+#### Basics of sorting
```

Comment:
I would simplify. For the end user, there's nothing special about sorting by user name or user role when compared to sorting by dashboard name. That's an implementation detail

Reply:
Maybe "special case" is not the correct term here, but I think this kind of sorting is different from the usual sorting we support, because these fields don't belong to the dashboard model. So if we don't state that this kind of sorting is supported, then it won't be documented  anywhere that we support them.

However, I agree that we might simplify this by just adding a line in the "Basics of sorting" section, instead of creating a whole new "Special sorting criteria" section for this.

Sounds good?

Reply:
not saying we shouldn't say we support sorting by user.name or user.role, I'm suggesting we don't make a distinction between sorting by those or by name.

Reply:
Simplified the docs for sorting, can you check again if this way is ok?